tests/sampling/test_jax.py

```diff
@@ -49,6 +51,26 @@ def test_old_import_route():
     assert set(new_sj.__all__) <= set(dir(old_sj))
 
 
+@pytest.mark.parametrize(
+    argnames="input, expected",
+    argvalues=[
+        (jnp.eye(3), True),
+        (jnp.zeros(shape=(10, 10)), False),
+        (jnp.array([[1, -1.5], [0, 1.2]], dtype="float32"), True),
+        (-1 * jnp.array([[1, -1.5], [0, 1.2]], dtype="float32"), False),
+        (jnp.array([[1, -1.5], [0, -1.2]], dtype="float32"), False),
+        (jnp.array([[1, 0.99, 1], [0.99, 1, 0.999], [1, 0.999, 1]], dtype="float32"), False),
+    ],
+)
+def test_jax_PosDefMatrix(input, expected):
```

Comment:
Let's not parametrize, it's quite wasteful to compile a function for each different value we want to test.

Reply:
So should l loop over the cases inside the test to test all the cases?

Reply:
Yup